README.md

```diff
@@ -33,6 +33,12 @@ If your .netrc file is not located in your home directory, you can specify its l
 export COCOAPODS_ART_NETRC_PATH=$HOME/myproject/.netrc
 ```
 
+You could set the following environment variable: COCOAPODS_ART_SSL_NO_REVOKE, this will add the flag --ssl-no-revoke to curl command.
+If your are running on an environment where access to CRL isn't available you would still be able to access Artifactory via HTTPS using the cocoapods-art plugin on windows.
+```
+set COCOAPODS_ART_SSL_NO_REVOKE=true
```

Comment:
Should this be `export` rather than `set`?

Reply:
It is only relevant on Windows which uses WinSSL for curl, therefore set is used (it is a windows command)

Reply:
Ah, that makes sense, it's been a long time since I've been a Windows dev. Sorry for the bother!

Reply:
Basically this flag would be rarely used as this art-cocoapods plugin is mainly about macOS development. But there are use cases such as ours where the security scanning for instance is being performed on an analyst's workstation which is windows based. He has all the tooling right now to scan Podfiles but he is unable to pull the Artifact since we are running on premise and access to CRL servers is blocked (no outbound traffic) so pulling pods with the plugin via HTTPS fails.
Hope the intentions here are clear enough.
Thank you for reviewing

Reply:
That's an interesting use case that I wouldn't have even thought of. Thanks for the explanation!

Reply:
@eyalbe4 done